bindings/ruby/examples/remote-probe.rb

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/ruby
+
+require 'storage'
+require 'open3'
+
+$host = "localhost"
+
+def run_command(name)
+  cmd = "ssh -l root #{$host} #{name}"
+  stdin, stdout, stderr, wait_thr = Open3.popen3(cmd)
+
+  ret = Storage::RemoteCommand.new
+
+  stdout.read.each_line { |line| ret.stdout << line.rstrip }
+  stderr.read.each_line { |line| ret.stderr << line.rstrip }
+
+  ret.exit_code = wait_thr.value.to_i
+
+  return ret
+end
+
+class MyRemoteCallbacks < Storage::RemoteCallbacks
```

Comment:
Maybe there should be a call to the constructor of the parent class Storage::RemoteCallbacks:

```
def initialize
    super
end
```